CrossPlatformToDoList/CrossPlatformToDoList/ToDo.m

```diff
@@ -10,10 +10,13 @@
 
 @implementation ToDo
 
-- (instancetype)initWithTitle:(NSString *)title withContent:(NSString *)content {
+- (instancetype)initWithTitle:(NSString *)title withContent:(NSString *)content withKey:(NSString *)key andCompleted:(NSNumber *)completed{
  
```

Comment:
When you create a custom initializer, the first thing you have to do is call `[super init]`.

A cleaner way of doing this would be:

```
- (instancetype)initWithTitle:(NSString *)title withContent:(NSString *)content withKey:(NSString *)key andCompleted:(NSNumber *)completed {
    self = [super init];

    if (self) {
        _title = title;
        ...etc, etc for the other properties.
    }
    return self;
}
```